javascript-source/core/commandCoolDown.js

```diff
@@ -95,76 +116,69 @@
      * @param  {String}  command
      * @param  {String}  username
      * @param  {Boolean} isMod
-     * @return {Number}
+     * @return {[Number, bool]}
      */
     function get(command, username, isMod) {
-        var cooldown = cooldowns[command];
+        var isGlobal = false,
+            maxCoolDown = 0;
+            
+
+        if(canIgnore(username, isMod)) {
+            return [maxCoolDown, isGlobal];
+        }
 
         if (isSpecial(command)) {
-            if (command == 'adventure' && defaultCooldowns[command] !== undefined && defaultCooldowns[command] > $.systemTime()) {
-                return defaultCooldowns[command];
-            } else {
-                return 0;
+            if (command.equalsIgnoreCase('adventure') && defaultCooldowns[command] !== undefined && defaultCooldowns[command] > $.systemTime()) {
+                maxCoolDown = getTimeDif(defaultCooldowns[command]);
+                return [maxCoolDown, isGlobal];
```

Comment:
Return is redundant

Reply:
Thanks ... a little oversight from a last minute cleanup I've made to improve the codeflow. I will commit these in a few hours